ast/node_convert.go

```diff
@@ -250,6 +254,34 @@ func (node *BinaryExprNode) handleStringOps() (BoolNode, error) {
 	return node.invalidOpTypes()
 }
 
+func (node *BinaryExprNode) handleCaseInsensitive(left, right StringNode) BoolNode {
+	op := BinaryOpContains
+	if node.op == BinaryOpNotIContains {
+		op = BinaryOpNotContains
+	}
+
+	return &BinaryStringExprNode{
```

Comment:
Doesn't `strings.EqualFold` already do case insesitivity?

Reply:
problem is that we're not doing equals, we're doing contains, and there's no built in containsfold